modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less

```diff
@@ -73,9 +73,41 @@
     text-transform: @toolbar-button-text-transform;
     width: @toolbar-button-width;
 
+    @media (forced-colors: active) {
+      &:hover {
+        filter: brightness(70%);
+      }
+
+      &.tox-tbtn--active,
+      &.tox-tbtn--enabled,
+      &.tox-tbtn--enabled:hover,
+      &.tox-tbtn--enabled:focus,
+      &:focus:not(.tox-tbtn--disabled) {
+        filter: brightness(50%);
```

Comment:
A technique we used in a previous editor was to add an outline for the hover state (similar to focus, but a lighter colour). Could we try that?

Reply:
yeah outline would work a lot better, would you say the same colour `#f0f` would work?

Reply:
I'm not sure what colour would work, sorry